src/relentless/simulate/hoomd.py

```diff
@@ -138,54 +132,7 @@ def _call_v3(self, sim):
                 dihedral_potential.params[i] = dict(U=u[:], tau=tau[:])
             sim[self]["_potentials"].append(dihedral_potential)
 
-    def _call_v2(self, sim):
-        # initialize
-        sim[self]["_system"] = self._initialize_v2(sim)
-        sim.dimension = sim[self]["_system"].box.dimensions
-        sim.types = sim[self]["_system"].particles.types
-
-        # parse masses by type
-        with sim["engine"]["_hoomd"]:
-            snap = sim[self]["_system"].take_snapshot(particles=True)
-            sim.masses = self._get_masses_from_snapshot(sim, snap)
-            self._assert_dimension_safe(sim, snap)
-
-        # attach the potentials
-        def _table_eval(r_i, rmin, rmax, **coeff):
-            r = coeff["r"]
-            u = coeff["u"]
-            f = coeff["f"]
-            return (numpy.interp(r_i, r, u), numpy.interp(r_i, r, f))
-
-        with sim["engine"]["_hoomd"]:
-            neighbor_list = hoomd.md.nlist.tree(
-                r_buff=sim.potentials.pair.neighbor_buffer
-            )
-            pair_potential = hoomd.md.pair.table(
-                width=len(sim.potentials.pair.linear_space), nlist=neighbor_list
-            )
-
-            r, u, f = sim.potentials.pair.pairwise_energy_and_force(
-                sim.types, tight=True, minimum_num=2
-            )
-            for i, j in sim.pairs:
-                if numpy.any(numpy.isinf(u[i, j])) or numpy.any(numpy.isinf(f[i, j])):
-                    raise ValueError("Pair potential/force is infinite at evaluated r")
-                pair_potential.pair_coeff.set(
-                    i,
-                    j,
-                    func=_table_eval,
-                    rmin=r[0],
-                    rmax=r[-1],
-                    coeff=dict(r=r, u=u[i, j], f=f[i, j]),
-                )
-
-    def _initialize_v3(self, sim):
-        raise NotImplementedError(
-            f"{self.__class__.__name__} not implemented in HOOMD {_hoomd_version}"
-        )
-
-    def _initialize_v2(self, sim):
+    def _initialize(self, sim):
         raise NotImplementedError(
             f"{self.__class__.__name__} not implemented in HOOMD {_hoomd_version}"
         )
```

Comment:
Make this an abstract method required to be overridden.
```suggestion
    @abc.abstractmethod
    def _initialize(self, sim):
        pass
```